gavel/templates/admin.html

```diff
@@ -22,9 +82,9 @@ <h2>Projects</h2>
   </div>
   <div class="info">
     <table id="items">
-      <thead>
+      <!-- <thead>
```

Comment:
To avoid having unexpected behaviours, I recommend to bring this commented code back to be as it was originally

Reply:
Removed!